apps/web/src/lib/components/home/HomeWelcome.svelte

```diff
@@ -3,25 +3,39 @@
 </script>
 
 <section
-	class="container mx-auto my-32 flex h-fit flex-col justify-center gap-8 px-4 md:my-64"
+	class="container mx-auto flex h-fit flex-col justify-center gap-8 px-4 md:my-64"
 >
-	<h2 class="h2 text-center text-4xl font-bold" data-testid="welcome-header">
-		Come join us!
-	</h2>
-	<p
-		class="mx-auto text-center text-xl md:max-w-[50rem]"
-		data-testid="welcome-subheader"
-	>
-		We welcome you to our worship services every Sunday morning at 10:30am!
-		Services will be both online (Zoom) and in-person. Click the button below to
-		join via Zoom, or join us in-person at
-		<a href={links.googleMaps} target="_blank" class="anchor font-bold">
-			5225 Orbitor Drive Unit 3, Mississauga, ON.
-		</a>
-	</p>
+	<div class=" w-full mt-10 text-white rounded-lg bg-slate-800 p-8">
```

Comment:
for this site, let's try to stick with a few colours that go with the colours of the logo.

so it's more green, blue, and white -- let's try not to use `slate-` and all these other colours so that we stay consistent